proto/regen/data/v1alpha2/query.proto

```diff
@@ -0,0 +1,152 @@
+syntax = "proto3";
+
+package regen.data.v1alpha2;
+
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "regen/data/v1alpha2/types.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/data";
+
+// Query is the regen.data.v1alpha1 Query service
+service Query {
+  // ByContentHash queries data based on its ContentHash.
+  rpc ByContentHash (QueryByContentHashRequest) returns (QueryByContentHashResponse);
+
+  // BySigner queries data based on signers.
+  rpc BySigner (QueryBySignerRequest) returns (QueryBySignerResponse);
+
+  rpc IDsByIRI (QueryIDsByIRIRequest) returns (QueryIDsByIRIResponse);
+
+  rpc IRIsByID (QueryIRIsByIDRequest) returns (QueryIRIsByIDResponse);
```

Comment:
in which context those might be useful given https://github.com/regen-network/regen-ledger/pull/221#issuecomment-768730796?

Reply:
They're utility methods is you want to build a CompactGraph in client code, which generally isn't needed but should be possible